index.bs

```diff
@@ -115,6 +115,10 @@ spec: HTML; urlPrefix: https://html.spec.whatwg.org/
 spec:infra; type:dfn; for:/; text:set
 </pre>
 
+<style>
+var {color: indianred}
```

Comment:
```suggestion
var {
  color: #cd5c5c
}
```

Reply:
By default the bikeshed generation creates colors in hex format. In case we want to use it across the board...